File: code_test/plusn.py
def plusn(n):
    result = 0
    for i in range(1, n+1):
        result += i

    return result


def plusn2(n):
    return n * (n+1) // 2


def multiplen(n):
    result = 0
    for i in range(1, n+1):
        result += i*i

    return result


def multiplen2(n):
    return n * (n + 1) * (2*n + 1) // 6


print(multiplen2(10))

File: code_test/test/plusn.ts
function plusn(n: number) {
  let result = 0;
  for (let i = 1; i < n + 1; i++) {
    result += i;
  }

  return result;
}

function plusn2(n: number) {
  return (n * (n + 1)) / 2;
}

console.log(plusn2(10));
